java/code/src/com/redhat/rhn/taskomatic/task/sshpush/test/SSHPushWorkerSaltTest.java

```diff
@@ -240,7 +247,7 @@ public void testSystemIsRebooting() throws Exception {
             }
         };
 
-        SSHPushWorkerSalt worker = successWorker(systemQuery);
+        SSHPushWorkerSalt worker = successWorker(saltService, saltService);
```

Comment:
This looks funny :)

Reply:
yeah it weird until we split the implementation of SystemQuery and SaltApi so they are not implemented by the same class.